libs/utils/utils.js

```diff
@@ -32,6 +32,7 @@ const MILO_BLOCKS = [
   'media',
   'merch',
   'modal',
+  'ost',
```

Comment:
This doesn't need to be here. No other site should implement the OST block.

Reply:
Removed ✅